assets/src/scripts/monitoring-location/components/hydrograph/index.test.js

```diff
@@ -318,10 +318,6 @@ describe('monitoring-location/components/hydrograph module', () => {
             expect(selectAll('#change-time-span-container').size()).toBe(1);
         });
 
-        it('should have download data form', () => {
```

Comment:
Why was this test removed? We still have a download-graph-data-container?

Reply:
Whoops.  My bad. At one point I modified this test. Then when removing the modification, I apparently got a little carried away with my deletions. Good catch!